istio-proxy/VERSION

```diff
@@ -0,0 +1 @@
+maistra-0.9
```

Comment:
Wonder if we can rely on the current branch name. i.e.:

git branch | grep \* | cut -d ' ' -f2

Reply:
We could, but I wanted to have branch name/version set explicitly: a pre-build check if you want.

Reply:
I can see a need for the ability to override to create test/debug versions based off of a different branch, but in terms of a release, I think we probably always want to make sure that the branches match up. I could go either way on this.